src/api/json/catalog.json

```
@@ -319,6 +319,15 @@
       "fileMatch": [".asyncapi-tool"],
       "url": "https://raw.githubusercontent.com/asyncapi/website/master/scripts/tools/tools-schema.json"
     },
+    {
+      "name": "Aurora Agile Meta-Framework",
+      "description": "Yaml schema for Aurora Agile Meta-Framework",
+      "fileMatch": ["*.aurora.yaml"],
```

```suggestion
      "fileMatch": ["*.aurora.yaml"],
```

Is `*.yml` extension allowed or just `*.yaml`?

---

Thanks! I have fixed it in the PR #3018